lib/lotus/lotusrc.rb

```diff
@@ -72,54 +84,23 @@ class Lotusrc
     # @see Lotus::Environment#initialize
     def initialize(root, options = {})
       @root    = root
-      @options = options
-
-      # NOTE this line is here in order to auto-upgrade applications generated
-      # with lotusrb < 0.3.0. Consider to remove it in the future.
-      create
+      @_options = options
     end
 
-    # Read lotusrc file and parse it's values.
+    # Read lotusrc file (if exists) and parse it's values or return default.
     #
     # @return [Lotus::Utils::Hash] parsed values
     #
     # @example Default values if file doesn't exist
-    #   Lotus::Lotusrc.new(Pathname.new(Dir.pwd)).read
+    #   Lotus::Lotusrc.new(Pathname.new(Dir.pwd)).options
     #    # => { architecture: 'container', test: 'minitest', template: 'erb' }
     #
     # @example Custom values if file doesn't exist
     #   options = { architect: 'application', test: 'rspec', template: 'slim' }
-    #   Lotus::Lotusrc.new(Pathname.new(Dir.pwd), options).read
+    #   Lotus::Lotusrc.new(Pathname.new(Dir.pwd), options).options
     #    # => { architecture: 'application', test: 'rspec', template: 'slim' }
-    def read
-      if exists?
-        Utils::Hash.new(
-          Dotenv::Parser.call(content)
-        ).symbolize!
-      end
-    end
-
-    private
-
-    # Create lotusrc file if exists
-    #
-    # @since 0.3.0
-    # @api private
-    #
-    # @see Lotus::Lotusrc::DEFAULT_ARCHITECTURE
-    # @see Lotus::Lotusrc::ARCHITECTURE_KEY
-    # @see Lotus::Lotusrc::DEFAULT_TEST_SUITE
-    # @see Lotus::Lotusrc::TEST_KEY
-    # @see Lotus::Lotusrc::DEFAULT_TEMPLATE
-    # @see Lotus::Lotusrc::TEMPLATE_KEY
-    def create
-      unless exists?
-        rcfile = File.new(path_file, "w")
-        rcfile.puts "#{ ARCHITECTURE_KEY }=#{ @options.fetch(:architecture, DEFAULT_ARCHITECTURE) }"
-        rcfile.puts "#{ TEST_KEY }=#{ @options.fetch(:test, DEFAULT_TEST_SUITE) }"
-        rcfile.puts "#{ TEMPLATE_KEY }=#{ @options.fetch(:template, DEFAULT_TEMPLATE) }"
-        rcfile.close
-      end
+    def options
```

Comment:
@aderyabin Can you please get back to `#read` instead of `#options`? The verb indicates the intent of reading the file.